frontend/src/pages/accounts/[id].tsx

```diff
@@ -14,31 +15,28 @@ import TransactionIconSvg from "../../../public/static/images/icon-t-transaction
 import { useTranslation } from "react-i18next";
 import { GetServerSideProps, NextPage } from "next";
 import { useAnalyticsTrackOnMount } from "../../hooks/analytics/use-analytics-track-on-mount";
-import { getPrefetchObject } from "../../libraries/queries";
-import { useQuery } from "../../hooks/use-query";
+import { trpc } from "../../libraries/trpc";
 import { styled } from "../../libraries/styles";
 import * as React from "react";
-import { useInfiniteQuery } from "../../hooks/use-infinite-query";
 
 const TransactionIcon = styled(TransactionIconSvg, {
   width: 22,
 });
 
-interface Props {
-  accountId: string;
-}
-
 const TRANSACTIONS_PER_PAGE = 10;
 
-const AccountDetail: NextPage<Props> = React.memo(({ accountId }) => {
+const AccountDetail: NextPage = React.memo(() => {
+  const accountId = useRouter().query.id as string;
```

Comment:
@frol @sheleged
JFYI we don't need to prefetch anything anymore - just use queries as you live in CSR world - the SSR prefetch is included.